code/iaas/storage-service/src/main/java/io/cattle/platform/storage/api/filter/ExternalTemplateInstanceFilter.java

```diff
@@ -71,6 +71,9 @@ else if (!StringUtils.isBlank(defaultRegistry) && !StringUtils.isEmpty(defaultRe
         if (image.startsWith(DockerImage.KIND_PREFIX)) {
             fullImageName = DockerImage.KIND_PREFIX + fullImageName;
         }
+        if (image.startsWith("sim:") && !fullImageName.startsWith("sim:")) {
+            fullImageName = "sim:" + fullImageName;
+        }
```

Comment:
we will strip `sim:` out, then add it back. This makes sure `storageService.isValidUUID` will return true.( it needs to have `docker:` or `sim` prefixed. Also avoid adding `sim` twice.